Member.java

```diff
@@ -0,0 +1,119 @@
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@SuppressWarnings("serial")
+public class Member implements Serializable {
+
+    private String lastName;
+    private String firstName;
+    private String email;
+    private int phoneNo;
+    private int id;
+    private double fines;
+    
+    private Map<Integer, loan> loans;
+
+    
+    public Member(String lastName, String firstName, String email, int phoneNo, int id) {
+        this.lastName = lastName;
+        this.firstName = firstName;
+        this.email = email;
+        this.phoneNo = phoneNo;
+        this.id = id;
+        
+        this.loans = new HashMap<>();
+    }
+
+    
+    public String toString() {
+        StringBuilder memberAttributes = new StringBuilder();
+        memberAttributes.append("Member:  ").append(id).append("\n")
+          .append("  Name:  ").append(lastName).append(", ").append(firstName).append("\n")
+          .append("  Email: ").append(email).append("\n")
+          .append("  Phone: ").append(phoneNo)
+          .append("\n")
+          .append(String.format("  Fines Owed :  $%.2f", fines))
+          .append("\n");
+        
+        for (loan loan : loans.values()) {
+            memberAttributes.append(loan).append("\n");
+        }          
+        return memberAttributes.toString();
+    }
+
+    
+    public int getID() {
+        return id;
+    }
+
+    
+    public List<loan> getLoans() {
+        return new ArrayList<loan>(loans.values());
+    }
+
+    
+    public int numberOfCurrentLoans() {
+        return loans.size();
+    }
+
+    
+    public double finesOwed() {
+        return fines;
+    }
+
+    
+    public void takeOutLoan(loan loan) {
+        int loanID = loan.ID();
+        if (!loans.containsKey(loanID)) {
+            loans.put(loan.ID(), loan);

```

Comment:
Please remove method within method call.
```suggestion
            loans.put(loanID, loan);
```